### ISSUE TITLE
Research Questions and aims

### ISSUE THREAD

**Issue by will-ball** [AbdnCHDS/covid-social-distancing#8]
Here's a starting point for defining our research questions and study aims. Please discuss and suggest improvements or alternative approaches.

***
### Working Title:
Validating the Grampian Social Distancing Score

### Brief Background:
A Social Distancing Index from [Colorado](https://www.coloradohealthinstitute.org/research/colorado-covid-19-social-distancing-index) was adapted (By Jess) and applied to the [Grampian Health Board](https://jessbutler.github.io/social_distancing/#social-distancing-score) using publically available ecological data for small areas.

The Index attempts to highlight areas where social distancing is more difficult.

### Research Question(s):
**V1**
~~1a) Does the Social Distancing Score for Grampian (SDS-G) predict COVID-19 testing demand?~~
~~1b) Does the SDS-G predict incidence of PCR positive COVID-19 cases?~~
~~2a) Does the SDS-G predict case fatality rates? (i.e. Daily reporting definition: Deaths within 28-days of +ve PCR test)~~
~~2b) Does the SDS-G predict certified COVID-19 death rates? (i.e. NRS definition: Deaths where COVID is mentioned on the death certificate)~~

**V2**
To what extent does the Social Distancing Score for Grampian (SDS-G) explain geographical variation in: 
1) COVID-19 testing demand?
2) PCR positive incidence?
3) Case fatality rates? (Death within 28 days of +ve PCR)
4) Certified COVID-19 mortality? (Death where COVID is mentioned on the death certificate)

**Comment by will-ball** [comment]
> 
> 1a) Does the Social Distancing Score for Grampian (SDS-G) predict COVID-19 testing demand?
> 1b) Does the SDS-G predict incidence of PCR positive COVID-19 cases?
> 2a) Does the SDS-G predict case fatality rates? (i.e. Daily reporting definition: Deaths within 28-days of +ve PCR test)
> 2b) Does the SDS-G predict certified COVID-19 death rates? (i.e. NRS definition: Deaths where COVID is mentioned on the death certificate)
>

An alternative thought on this, perhaps guided by statistical adivce, would be _'To what extent does the SDS-G explain geographical variation in X'_

**Comment by BScheliga** [comment]
> > 1a) Does the Social Distancing Score for Grampian (SDS-G) predict COVID-19 testing demand?
> > 1b) Does the SDS-G predict incidence of PCR positive COVID-19 cases?
> > 2a) Does the SDS-G predict case fatality rates? (i.e. Daily reporting definition: Deaths within 28-days of +ve PCR test)
> > 2b) Does the SDS-G predict certified COVID-19 death rates? (i.e. NRS definition: Deaths where COVID is mentioned on the death certificate)
> 
> An alternative thought on this, perhaps guided by statistical adivce, would be _'To what extent does the SDS-G explain geographical variation in X'_

@will-ball 
I have been wondering the same, I think that might be better wording at the moment. As it feels to me, when we use the word "predict" our results should be a model that predicts X using the SDS-G.

**Comment by dblana** [comment]
Brilliant start! I agree with you both @will-ball @BScheliga that we are not looking to predict X from SDS-G, but see if there is a correlation. Quick question: what do we mean by "testing demand" (1a)? I'll tag here @gosler01 so he knows what we are doing and see if he has any comments 😃

**Comment by will-ball** [comment]
Agreed. Questions updated in first comment.

By 'testing demand' I meant total tests (both positive and negative) - Looking at that I think allows us to explore overall demand & access to testing by area, as well as test positivity. It's maybe not essential but if we have the negative test results also it could be quite revealing.

**Comment by BScheliga** [comment]
Sorry, I have a third suggestion. But, I not 100 % convinced with my following rational. See what you think.

The SDS-G for our analysis is static in time, but varies in space. For the Covid related variables (CRV), if we are looking at the total numbers it is the same. Hence, you could say: we are trying to understand the spatial relationship between SDS-G and CRV. 

Now, if we start looking at the difference between the 1. & 2. Covid wave, we are introduction a temporal component to our spatial analysis. Hence, our aims or research objectives would be: 
“Trying to understanding the (dynamics* of) spatio-temporal relationship between the SGS-G and the CVR”

*maybe “dynamics” is a bit overboard 

What you you reckon?

**Comment by will-ball** [comment]
That's an important point. Consistency in interpreting our findings would be majorly impaired if we just looked at this in an 'overall' sense.

I think there are three distinct time periods:
1) 1st wave - initially characterised by more limited testing, likely a large underestimate of 'true' positives
2) Interim - wider testing available but lower number of cases (assumption here as a result of lockdown/social distancing)
3) 2nd wave - wider testing availability, variable restrictions

Do we want to look at 1 vs. 2 vs. 3, or should we assess the relationship between SDS-G and CVR _within_ each of these distinct periods?

@dblana - do you have a sense of which time periods we have data for?

**Comment by dblana** [comment]
I agree that we shouldn't just look at the overall number. We should have data (positive + negative tests, deaths) from the start (last March) until now, so we can look at any time periods we want. The three periods you identify @will-ball make sense, but we could also ask Graham, Shantini, etc. for their opinion on that. When you say 1 vs. 2, what is it that we would be comparing there?

**Comment by will-ball** [comment]
Fantastic - It would be good to hear some more external opinions.

1 vs. 2 vs. 3 would refer to a comparison between the different time periods - i.e. a question like 'when is the SDS-G most correlated to outcomes?'

Personally, I'm hesitant to go down that road, especially if we focus on testing as we know there are quite big differences in things like the availability of testing.
